automation/project/Dependencies.scala

```diff
@@ -7,9 +7,9 @@ object Dependencies {
   val akkaV = "2.5.7"
   val akkaHttpV = "10.0.10"
 
-  val workbenchModelV  = "0.12-db13bb0"
+  val workbenchModelV  = "0.12-fe70cc3-SNAP"
```

Comment:
do you need to update this?

Reply:
Yeah, I'll have to update it once I get the workbench-libs PR merged, but wanted to get the tests in review as well just to get some feedback on them in the meantime.